.github/workflows/grab_spotify_saved_tracks.yaml

```
@@ -0,0 +1,42 @@
+            - name: Install the latest version of uv
+              uses: astral-sh/setup-uv@v3
+              with:
+                  version: "latest"
+                  enable-cache: true
+
+            - name: Run spotify tracks script
+              working-directory: ./tools
+              run: uv run grab_spotify_tracks.py
+
+            - name: Commit and push if changes
+              run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2015:info:4:18: Note that A && B || C is not if-then-else. C may run when A is true

(shellcheck)